src/Cell.cpp

```diff
@@ -97,13 +97,23 @@ Cell::Cell(int id, const char* name) {
  */
 Cell::~Cell() {
 
+  // DUPLICATE of region deletion
   std::map<int, surface_halfspace*>::iterator iter;
+  std::map<int, surface_halfspace*> _surfaces = getSurfaces();
```

Comment:
Why do you call the getSurfaces function to return rather than use the _surfaces variable directly?

Reply:
Good catch, I should do that